Messerli.Session.AspNetCore/Internal/Response.cs

```diff
@@ -40,35 +41,27 @@ public void SetCookie(Cookie cookie)
         }
 
         public void SetHeader(string name, string value)
-        {
-            _httpContext.Response.Headers.Append(name, value);
-        }
+            => _httpContext.Response.Headers.Append(name, value);
 
-        public bool HasHeader(string name)
-        {
-            return _httpContext.Response.Headers.ContainsKey(name);
-        }
+        public string? GetFirstHeaderValue(string name)
+            => _httpContext.Response.Headers[name].FirstOrDefault();
```

Comment:
Add a comment explaining why this doesn't throw

Reply:
https://github.com/messerli-informatik-ag/session/pull/30/commits/1712aa42773e9586e5ef22ceab3355182b6a7899